Greenshot/Drawing/Adorners/AbstractAdorner.cs

```diff
@@ -116,6 +116,23 @@ public virtual Rectangle Bounds
 			}
 		}
 
+		/// <summary>
+		/// Return the bounds of the Adorner as displayed on the parent Surface
+		/// </summary>
+		protected virtual Rectangle SurfaceBounds

```

Comment:
I'm not quite happy with the naming here, as it would suggest that these are the bounds of the surface itself. Maybe BoundsOnSurface fits better??

Reply:
I agree